### PR TITLE
PD-79 Consent tracking

### PR DESCRIPTION
**Pull request by edwardupton**

### CODE REVIEW

File: src/common/helpers/getTrackingConsent.ts [hunk 1]
```diff
@@ -5,18 +5,29 @@ declare let window: CustomWindow;
...
+
+		window.Shopify.loadFeatures(
+			[
+				{
+					name: CONSENT_TRACKING_API,
+					version: '0.1',
+				},
+			],
+			(error: any) => {
+				if (error) return resolve(false);
```
Comment (lyykfi) [inline src/common/helpers/getTrackingConsent.ts:26]: wdyt to add tests here?

**Comment by lyykfi** [comment]
could you rebase by staging please to fix problems with tests?

File: src/common/helpers/getTrackingConsent.ts (hunk 1 above)
Comment (edwardupton) [reply src/common/helpers/getTrackingConsent.ts:26]: I already did:
https://github.com/littledata/shopify-tracker/pull/260/files#diff-115a49c1b0959ee1ffc1bb23ca79d3d6d9d6523fea40329c69437ccbbc071f9fR23

**Comment by lyykfi** [comment]
@edwardupton have you regenerated dist/*.js files?

**Comment by edwardupton** [comment]
No, but I updated from master with /dist folder changes. I've reverted that

**Review by lyykfi — APPROVED** [review]

**Comment by lyykfi** [comment]
Thank. I just though that there was a potential problem with the /dist folder.
approved

**Pull request merged**